helpdesk_mgmt_tickets_menu/views/helpdesk_mgmt_tickets_menu.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="action_my_helpdesk_tickets" model="ir.actions.act_window">
+        <field name="name">My Tickets</field>
+        <field name="res_model">helpdesk.ticket</field>
+        <field name="view_mode">kanban</field>
```

Comment:
We should add other available view modes. `kanban` should be the default one, but we could still select another one